scripts/compare_bencher_ab.py

```diff
@@ -0,0 +1,249 @@
+    is_improvement = (change_val > 0 and higher_is_better) or (
+        change_val < 0 and not higher_is_better
+    )
+
+    if abs(change_val) < 2:  # No significant change
+        bar = " " * center + "|" + " " * center
+        return bar + f" {change_val:+.1f}%"
+    elif is_improvement:  # This is actually an improvement
+        if change_val > 0:  # Positive change that's good (e.g., higher throughput)
+            bar = " " * center + "|" + "▓" * bar_length + " " * (center - bar_length)
+        else:  # Negative change that's good (e.g., lower latency)
+            left_start = center - bar_length
+            bar = " " * left_start + "▓" * bar_length + "|" + " " * center
+        return bar + f" {change_val:+.1f}%"
+    else:  # This is a regression
+        if change_val > 0:  # Positive change that's bad (e.g., higher latency)
+            bar = " " * center + "|" + "█" * bar_length + " " * (center - bar_length)
+        else:  # Negative change that's bad (e.g., lower throughput)
+            left_start = center - bar_length
+            bar = " " * left_start + "█" * bar_length + "|" + " " * center
+        return bar + f" {change_val:+.1f}%"
+
+
+def calculate_percentage_change(val1: float, val2: float) -> str:
+    """Calculate percentage change from val1 to val2"""
+    if val1 is None or val2 is None:
+        return "N/A"
+    if val1 == 0:
+        return "∞" if val2 != 0 else "0%"
+
+    change = ((val2 - val1) / val1) * 100
+    sign = "+" if change > 0 else ""
+    return f"{sign}{change:.1f}%"
+
+
+def create_side_by_side_plot(
+    file1: str, file2: str, label1: str = None, label2: str = None
+):
+    """Create ASCII side-by-side comparison plot"""
+
+    # Default labels
+    if not label1:
+        label1 = file1.replace(".json", "")
+    if not label2:
+        label2 = file2.replace(".json", "")
+
+    # Load data
+    data1 = load_bencher_file(file1)
+    data2 = load_bencher_file(file2)
+
+    # Extract metrics
+    metrics1 = extract_metrics(data1)
+    metrics2 = extract_metrics(data2)
+
+    # Get all unique metric keys
+    all_keys = set(metrics1.keys()) | set(metrics2.keys())
+    all_keys = sorted(all_keys)
+
+    if not all_keys:
+        print("No metrics found in the files")
+        return
+
+    # We don't need global min/max anymore since we normalize each metric independently
+
+    # Print header
+    total_width = 120
+    print("=" * total_width)
+    title = f"BENCHMARK COMPARISON: {label1} vs {label2}"
+    print(f"{title:^{total_width}}")
+    print("=" * total_width)
+    print()
+
+    # Column widths
+    metric_width = 40
+    bar_width = 50  # Width for the diverging bar chart
+    values_width = 30
+
+    # Print column headers
+    print(
+        f"{'Metric':<{metric_width}} {'Performance Change':^{bar_width}} {'Values':^{values_width}}"
+    )
+    print(
+        f"{'':<{metric_width}} {'← Better  |  Worse →':^{bar_width}} {label1 + ' → ' + label2:^{values_width}}"
```

Comment:
Since "Better or Worse" is now determined per-metric, we should remove this header from the column.